addons/pinniped/config-controller/controllers/suite_test.go

```diff
@@ -80,8 +80,8 @@ var _ = BeforeSuite(func() {
 	mgr, err := ctrl.NewManager(testEnv.Config, options)
 	Expect(err).NotTo(HaveOccurred())
 
-	reconciler := NewController(k8sClient)
-	Expect(reconciler.SetupWithManager(mgr)).To(Succeed())
+	Expect(NewV1Controller(k8sClient).SetupWithManager(mgr)).To(Succeed())
```

Comment:
praise: i think i like that we are running these tests with both controllers, makes for a stronger test